tests/ingest_runs/test_ingest_runs_mutations.py

```diff
@@ -91,6 +92,35 @@ def test_start_ingest_run(
         assert resp.json()["errors"][0]["message"] == "Not allowed"
 
 
+@pytest.mark.parametrize(
+    "user_group",
+    [
+        ("Administrators"),
+    ],
+)
+def test_start_ingest_run_event(
+    db, clients, prep_file, mock_ingest_job, user_group
+):
+    """
+    Test that events are emitted correctly.
+    """
+    client = clients.get(user_group)
+
+    for i in range(2):
+        prep_file(authed=True)
+    version_ids = [
+        to_global_id("VersionNode", f.versions.first().kf_id)
+        for f in File.objects.all()
+    ]
+
+    # Start valid ingest runs for a batch of file versions
+    resp = send_query(client, START_INGEST_RUN, {"versions": version_ids})
+
+    assert Event.objects.count() == 1
```

Comment:
Right idea, but I a little extraneous since we already have a test for the start ingest run mutation. Could you move the event checking block into the `test_start_ingest_run` test in the `if allowed` block?